app/assets/stylesheets/views/_covid.scss

```diff
@@ -137,6 +137,47 @@ $covid-grey: #272828;
   margin-bottom: govuk-spacing(2);
 }
 
+.covid__video-placeholder {
+  position: relative;
+  padding: govuk-spacing(9) 0;
+  border: solid 2px govuk-colour("white");
+  text-align: center;
+
+  .govuk-body {
+    margin: 0;
+  }
+
+  .covid__bold {
+    font-weight: bold;
+  }
+}
+
+.covid__video-logo {
+  @include govuk-font(19, $weight: bold, $line-height: 1);
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: govuk-spacing(1);
+  padding-top: govuk-spacing(2);
+  padding-left: govuk-spacing(5);
+  padding-right: govuk-spacing(2);
+  margin: govuk-spacing(3);
+  border: solid 1px $covid-green;
+  color: govuk-colour("white");
+  text-transform: uppercase;
+
+  &:before {
+    content: "";
+    position: absolute;
+    top: 11px;
+    left: 8px;
+    width: 12px;
+    height: 12px;
+    background: $covid-green;
```

Comment:
Same comment about updating to $covid-yellow

Reply:
Also updated.